File: Java/Day5Loops.java
import java.io.*;

public class Solution {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));

        int n = Integer.parseInt(bufferedReader.readLine().trim());
        
        for (int i = 1; i <= 10; i++) {
        	int multiples = n * i;
            System.out.println(n + " x " + i + " = " + multiples);
        }

        bufferedReader.close();
    }
}

File: C#/Day5Loops.cs
﻿using System;

class Solution
{
    public static void Main(string[] args)
    {
        int n = Convert.ToInt32(Console.ReadLine().Trim());

        for (int i = 1; i <= 10; i++)
        {
            int multiples = n * i;
            Console.WriteLine(n + " x " + i + " = " + multiples);
        }
    }
}
